### DISCUSSION TITLE
Welcome to iris-esmf-regrid Discussions!

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris-esmf-regrid#43] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bjlittle** [comment]
@zklaus @stephenworsley and @jamesp 

Perhaps `GitHub Discussions` might help with remote collaboration... give it a whirl and see if it helps :smile:

**Comment by dennissergeev** [comment]
Hi all. I have a small question so I'm posting it here instead of the issues page, I hope it's ok. 
Is there a reason the Python version is capped at 3.8 for this library? It would be great to relax it to allow for the latest versions, 3.9 and 3.10, I think.

**Reply by stephenworsley** [reply]
Hi @dennissergeev,  I believe the current reason for capping the python version is due to the testing set up inheriting from iris where testing is also not done on 3.9 due to problems with graphics tests. This is something we're looking at resolving.
